snippets/fish.json

```diff
@@ -0,0 +1,190 @@
+{
+  "shebang": {
+    "prefix": "shebang",
+    "description": "shebang",
+    "body": "#!/usr/bin/env fish"
+  },
+  "if operator": {
+    "prefix": "if",
+    "description": "if operator",
+    "body": [
+      "if ${1:condition}",
+      "\t$0",
+      "end"
+    ]
+  },
+  "if else operator": {
+    "prefix": "if-else",
+    "description": "if operator",
+    "body": [
+      "if ${1:condition}",
+      "\t${2:echo}",
+      "else",
+      "\t$0",
+      "end"
+    ]
+  },
+  "else operator": {
```

Comment:
The else snippet can probably be omitted since the `if` and `if else` are already defined here